snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/FileBasedSnapshotStoreFactory.java

```diff
@@ -86,4 +87,15 @@ public ReceivableSnapshotStore getReceivableSnapshotStore(final int partitionId)
   public PersistedSnapshotStore getPersistedSnapshotStore(final int partitionId) {
     return partitionSnapshotStores.get(partitionId);
   }
+
+  /**
+   * Return the same concurrent control (actor) that is used by the snapshot store of the given
+   * partition
+   *
+   * @param partitionId
+   * @return concurrency control
+   */
```

Comment:
:wrench: I would mark it as deprecated already to discourage usage, and we should work on refactoring this soon-ish